src/bin/solang.rs

```diff
@@ -156,6 +156,12 @@ fn main() {
                 .long("math-overflow")
                 .display_order(5),
         )
+        .arg(
+            Arg::with_name("COMMONEXPRESSIONELIMINATION")
+                .help("Disable common subexpression elimination")
+                .long("no-cse")
+                .display_order(6),
+        )
```

Comment:
--no-cse should be ordered before --math-overflow, please set the display order to 5 (and move math-overflow below this)

Reply:
Done